codecov.yml

```diff
@@ -0,0 +1,10 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  status:
+    project:
+      default: false  # disable the default status that measures entire project
+    patch:
+      default:
+        target: 75%  # new contributions should have a coverage at least equal to target
```

Comment:
@hedgar2017 , do you think 75% coverage for the new code is a good number? Or do you prefer to lower the number, or maybe disable it for now?

Reply:
@antonbaliasnikov please disable completely.
It is only needed for informational purposes, e.g. to check if some important pieces are covered, not to force someone to write unit tests.

Reply:
It is actually more tricky than that.
Almost all our code can only be tested via `compiler-tester`, so if we wanted it enforce, it would only make sense to do it there. If it was possible at all this way.

Reply:
Done, disabled all checks.